frontend/src/utilities/__tests__/addTypesToK8sListedReources.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { addTypesToK8sListedResources } from '~/utilities/addTypesToK8sListedResources';
+
+const servingRuntimeTemplate = {
+  apiVersion: 'template.openshift.io/v1',
+  items: [
+    {
+      apiVersion: 'serving.kserve.io/v1alpha1',
+      kind: 'ServingRuntime',
```

Comment:
But this is not a k8s list, is a part of the template items, they can have kind and so, we even store it, so I think it's ok to have it

Reply:
This mock is being used in the tests as if it were a list of resources because of the `items` array.
The actual response of a call to `/api/templates/opendatahub?labelSelector=opendatahub.io%2Fdashboard%3Dtrue` is as follows, where the items contains an array of `objects` which are of kind `ServingRuntime`. But each entry within `items` does not have `apiVersion` or `kind`.
```
{
    "kind": "TemplateList",
    "apiVersion": "template.openshift.io/v1",
    "metadata": {
        "resourceVersion": "59077671"
    },
    "items": [
        {
            ...
            "objects": [
                {
                    "apiVersion": "serving.kserve.io/v1alpha1",
                    "kind": "ServingRuntime",
                    ...
                }
            ]
        },
        {...}
```

Reply:
Ohhh right, my bad, I got confused between `items` and `objects`. Then I'll say that the object itself is wrong, if it's called `servingRuntimeTemplate` I would assume it's a template that contains a `ServingRuntime` object, so the code would be

```javascript
const servingRuntimeTemplate = {
  apiVersion: 'template.openshift.io/v1',
  objects: [
    {
      apiVersion: 'serving.kserve.io/v1alpha1',
      kind: 'ServingRuntime',
...
```